app/build.gradle

```diff
@@ -188,7 +188,7 @@ dependencies {
   testImplementation(
       'androidx.test:core:1.2.0',
       'androidx.test.espresso:espresso-contrib:3.1.0',
-      'androidx.test.espresso:espresso-core:3.2.0',
+      'androidx.test.espresso:espresso-core:3.5.0-alpha03',
```

Comment:
It looks like 3.5.0 has been fully launched--could we use that instead of an alpha release?

Reply:
One of the depenency [changes](https://developer.android.com/jetpack/androidx/releases/test#espresso-3.5.0) introduced in 3.5.0 is to compile Espresso with kotlin stdlib 1.7.10, while our project uses 1.4.20.

I'm reluctant to update the kotlin version because I'm not sure what other breaking changes I might introduce.